src/app/workflow/dag/dag.component.html

```diff
@@ -48,18 +48,20 @@
                 (descriptorType$ | async) === ToolDescriptor.TypeEnum.CWL &&
                 (workflow$ | async)?.source_control_provider === 'GITHUB'
               "
-              style="line-height: 20px;"
+              style="line-height: 20px"
```

Comment:
Did prettifier do this? I always thought the semi-colon was proper, although browsers would typically let  you get away without it. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style) example has semi-colon.

Reply:
yeah, it's getting changed when I make a commit.

Reply:
Looked deeper and I was wrong. [CSS spec](https://drafts.csswg.org/css-style-attr/#style-attribute) doesn't have a trailing semi-colon.